README.md

```diff
@@ -33,3 +33,126 @@ Yeep provides a simple, open-source, privacy-first solution to user management.
 ## License
 
 Apache 2.0
+
+
+## Configuring the mail client
+
+Adding configuration options for the email service is required.
+We are using [nodemailer](https://nodemailer.com) v4+ under the hood so we will be supporting a bunch of different options out of the box. We currently support
+
+* SMTP
+* Amazon SES
+* Sendgrid
+* Mailgun
+* Debug
+
+An example object of the mail configuration as needed in the yeep.config.js file is
+```js
+{
+  transport: 'smtp',
+  from: 'admin@yeep.com',
+  // or you can use this format
+  // from: "'Custom Name' <myemail@address.com>",
+  templatePath: './path/to/email/templates' // if you want to provide custom email templates
```

Comment:
Was thinking about that too. But we don't really have to enforce copy-pasting of files missing. If they don't have the given file we can just fallback to the default one. But this might mean extra file checks on startup. I don't have strong feelings on this one

Reply:
So what you are saying is that our hypothetical dev would create a custom _reset-password_ email template, e.g. `./my-templates/reset-password.html` and then set the template path as `templatePath: './my-templates'`.

Any templates not found in the custom template path would revert to the default template directory path. I see a couple of issues with this approach...

1. The user is required to use the exact naming we provide, i.e. `reset-password.html`;
2. The user is required to put all templates in the same folder;
3. Template paths are implied, instead of specifically defined.

I could live with constraints 1-2, although I do find them unnecessary. The 3rd constraint is what I find most problematic. I prefer the config file to be as detailed as possible.